util/install_linux.sh

```diff
@@ -1,18 +1,28 @@
 #!/bin/bash
 
-# ensure that ~/.local/share/fonts exists
-mkdir -p ~/.local/share/fonts
+# Set the current user
+USER=$(logname)
+echo "Current user is $USER"
 
-# remove all fonts from ~/.local/share/fonts that start with "Monaspace"
-rm -rf ~/.local/share/fonts/Monaspace*
+# Set the target font directory
+TARGET_DIR="/home/$USER/.local/share/fonts/Monaspace"
```

Comment:
Should be `$HOME`.
Also, uppercase variables are usually used for environment variables, not local script variables.